### ISSUE TITLE
HTML5 export templates not found

### ISSUE THREAD

**Issue by tofuhead** [godotengine/godot#13435]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Kubuntu 17.10, Godot 3.0 Beta - Build 1 (and git master)

**Issue description:**
Installed the templates that come with Beta build1, all the other ones seem to work fine, but when I try to add html5 export it complains the export templates cannot be found. 

javascript_release.zip and javascript_debug.zip seem to be in correct directory, **~/.local/share/godot/templates/3.0-beta** 
I've tried self compiled versions, and the ones from here https://downloads.tuxfamily.org/godotengine/3.0/beta1/

.b

**Comment by akien-mga** [comment]
Yeah that's a naming issue, they should actually be named `webassembly_release.zip` and `webassembly_debug.zip`. I've uploaded a fixed export templates archive on https://downloads.tuxfamily.org/godotengine/3.0/beta1/, but you can just rename yours as mentioned above and it should work.

**Comment by Sseriously** [comment]
**ERROR**: still waiting on run dependencies:
**ERROR**: dependency: wasm-instantiate
**ERROR**: (end of list)
**ERROR**: pre-main prep time: 12037 ms
**ERROR**: WebGL 2 not available

**Comment by akien-mga** [comment]
That's unrelated to this issue, please open a new issue and provide actual information about your system, what you're doing, what is not working, and not just error messages.

**Comment by Sseriously** [comment]
Sorry Bro I'm a only 13 years old jk....i'm 29
